.gitignore

```diff
@@ -0,0 +1,20 @@
+# osx noise
+.DS_Store
+profile
+
+# xcode noise
+build/*
+*.mode1
+*.mode1v3
+*.mode2v3
+*.perspective
+*.perspectivev3
+*.pbxuser
+xcuserdata
+
+# svn & cvs
+.svn
+CVS
```

Comment:
Yeah... don't want those in there! :-) I think it's been 10 years or so since I last used Subversion.

Reply:
: ) I copied the gitignore from the tiqr repo